packages/esm-patient-registration-app/src/patient-registration/patient-registration.resource.tsx

```diff
@@ -6,6 +6,7 @@ import { Patient, Relationship, PatientIdentifier, Encounter } from './patient-r
 
 export const uuidIdentifier = '05a29f94-c0ed-11e2-94be-8c13b969e334';
 export const uuidTelephoneNumber = '14d4f066-15f5-102d-96e4-000c29c2a5d7';
+export const unidentifiedPatientAttributeTypeUuid = '8b56eac7-5c76-4b9c-8c6f-1deab8d3fc47';
```

Comment:
Seems weird to put a constant here that isn't used in this file. Just put it in the file we actually need it.

Reply:
Since this is the resource file, and this variable is being used in multiple places, hence I felt this okay to export from the resource file.

Reply:
This export is only used in the name-field component (unless I’m missing something). If it was used in multiple places, I could see moving it here. PS “resource” largely just means “api calls”; including constants used in the api calls but needed elsewhere as exports makes some sense.

Reply:
Got it. I remembered the UUID was being in multiple files, but forgot I cleaned it up as well.
I'll get to it.
Thanks!